provision/virtualbox.sh

```diff
@@ -3,5 +3,6 @@
 set -e
 set -x
 
-sudo pacman -S --noconfirm virtualbox-guest-utils-nox virtualbox-guest-modules-arch
+sudo pacman -Syy
+yes | sudo pacman -Syy virtualbox-guest-utils-nox virtualbox-guest-modules-arch
```

Comment:
I don't see a reason for substituting the line before.. `pacman -S --noconfirm` should be the same as `yes | pacman -S`.
Also why do you want to clean the package cache and redownload the fresh package base via `pacman -Syy`?